components/suggestion/suggestion_box.jsx

```diff
@@ -189,9 +196,11 @@ export default class SuggestionBox extends React.PureComponent {
             selection: '',
             allowDividers: true,
             presentationType: 'text',
+            tags: [],
```

Comment:
Does this component actually need to keep track of its tags? This seems like it might be left over code.